propra_webscience_ws24/training/model/svm.py

```diff
@@ -0,0 +1,12 @@
+from sklearn.svm import LinearSVC
+from propra_webscience_ws24.training.model.model_base import ModelBase, ModelType
+
+class SVM(ModelBase[LinearSVC]):
+
+    def _get_model(self, model_args) -> LinearSVC:
+        if "random_state" not in model_args:
+            model_args["random_state"] = 42
```

Comment:
:pick: Die Default-Werte könnten auch mit Aufrufen der `setdefault` Methode gesetzt werden (würde die `if`-Bedingungen auflösen):
```suggestion
        model_args.setdefault("random_state", 42)
```
Oder Du könntest den Dictionary Merge Operator `|` verwenden, mit welchem mehrere `if`-Bedingungen aufgelöst werden könnten:
```python
    d = {'foo': 42}

    # linker Ausdruck würde Default-Werte enthalten
    d = {'bar': 1, 'foo': 50} | d #  => d = {'bar': 1, 'foo': 42}

```

Reply:
Der zweite Ansatz sagt mir persönlich mehr zu.